sbc_compassion/models/correspondence_template.py

```diff
@@ -443,7 +398,7 @@ def _verify_template(tpl):
     """
     width = tpl.page_width
     height = tpl.page_height
-    if tpl.template_image and tpl.pattern_image:
```

Comment:
You can remove the whole method. Remove also the caller method `def _verify_template`